keps/sig-storage/3756-volume-reconstruction/kep.yaml

```diff
@@ -5,7 +5,7 @@ authors:
 owning-sig: sig-storage
 participating-sigs:
   - sig-node
-status: implementable
+status: implemented
```

Comment:
I wonder if we should merge this change after the implementation is complete.

Reply:
Correct, merge this when all is finished to avoid incorrect KEP if there is an implementation delay. After merging you can also close the underlying issue.